.github/workflows/dos-djgpp.yml

```diff
@@ -39,7 +43,7 @@ jobs:
       - name: Setup build environment
         run: |
           sudo apt -y update
-          sudo apt -y install build-essential cmake wget 7zip git flex nasm libslang2-dev pkg-config libslang2-modules gcc-multilib
+          sudo apt -y install build-essential cmake wget 7zip git flex nasm libslang2-dev pkg-config libslang2-modules gcc-multilib g++
```

Comment:
isn't `g++` part of `build-essential`?

Reply:
No.

Reply:
https://packages.ubuntu.com/mantic/build-essential

`g++` is definitely a dependency of `build-essential`

![image](https://github.com/UMSKT/UMSKT/assets/321592/7797bad5-0b97-466d-922b-00619f718901)

Reply:
But it doesn't install in it.